ts/details.ts

```diff
@@ -57,15 +50,15 @@ const tabBar = new TabBar(
 
 tabBar.build();
 
-// reload
-function reload() {
-  tabBar.tables.forEach((tbl, idx) => {
+// reload: destroy current table and build new table
+function reload(definition: TableDefinition<any, any>) {
```

Comment:
This function should only destroy and re-create the table. Tab-switching logic should be in `tab-bar-builder.ts`